api/src/main/java/com/epam/pipeline/controller/pipeline/PipelineRunController.java

```diff
@@ -595,4 +595,18 @@ public Result<List<RunInfo>> loadRunsByParentId(@PathVariable(RUN_ID) final Long
     public Result<RunChartInfo> loadActiveRunsCharts(@RequestBody final RunChartFilterVO filter) {
         return Result.success(runApiService.loadActiveRunsCharts(filter));
     }
+
+    @PostMapping("/run/{runId}/network")
+    @ApiOperation(
+            value = "Set limit boundary",
+            notes = "Sets a special tag for a run based on boundary param: NETWORK_LIMIT: <boundary> (Bytes/s) " +
+                    "in case of limit = true, otherwise removes the tag.",
+            produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)})
+    public Result<Boolean> setLimitBoundary(@PathVariable(value = RUN_ID) final Long runId,
+                                         @RequestParam(defaultValue = "true") final Boolean limit,
```

Comment:
'limit' -> 'enable' since we will have `limit` in the name of the method. Don't forget to change @ApiOperation notes